ui/react-components/components/Print/Print.module.scss

```diff
@@ -0,0 +1 @@
+@import url(../../../node_modules/font-awesome/css/font-awesome.min.css)
```

Comment:
is it possible to just load font in storybook? Instead of the main component style?

Reply:
I'm not sure how I should go about importing the font-awesome in one place that allows it to be used across all scss. I tried looking at the variables.scss but that is also imported in each and every component style

Reply:
I have added font-awesome icons required in the webpack config.js file. So we do not need to have any specific imports in the component specific files. Please check it out. And if all okay, then merge?